classes/03class/exercises/c03-docker05/gepifanio/README.md

```diff
@@ -0,0 +1,48 @@
+# C03-Docker05
+
+## Command Execution Output
+- Commands executed to run both containers:
+```
+docker run guiepifanio/exercise02:v1
+
+Unable to find image 'guiepifanio/exercise02:v1' locally
+v1: Pulling from guiepifanio/exercise02
+aad63a933944: Pull complete
+29ade582b51e: Pull complete
+7e41ad5b6f9c: Pull complete
+ebf61b47b4ca: Pull complete
+9c060bce4eae: Pull complete
+3be9cbb33c6f: Pull complete
+Digest: sha256:1a8773da5c34898fbf2b31bcf92f9d30fe87799463f0423444b76aadabb9dd6d
+Status: Downloaded newer image for guiepifanio/exercise02:v1
+AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
+AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
+[Wed Jul 15 13:49:17.506256 2020] [mpm_event:notice] [pid 1:tid 140186729925960] AH00489: Apache/2.4.41 (Unix) configured -- resuming normal operations
+[Wed Jul 15 13:49:17.506318 2020] [core:notice] [pid 1:tid 140186729925960] AH00094: Command line: 'httpd -D FOREGROUND'
+
+docker run guiepifanio/exercise02:v2
```

Comment:
Missing the output for this command 😢

Reply:
Oh true, haha forgot to paste it here :/